concrete/src/Package/StartingPointPackage.php

```diff
@@ -416,9 +416,10 @@ protected function install_api()
     {
         $scopes = $this->app->make('config')->get('app.api.scopes');
         $em = $this->app->make(EntityManager::class);
-        foreach($scopes as $scope) {
+        foreach($scopes as $scope => $description) {
```

Comment:
I see that this file was already out of compliance before this change, but I just wanted to mention it since this line is modified and not fixed. PSR-2 requires a space between `foreach` and `($scopes...`